src/ds/ts/retrieve.md

```diff
@@ -0,0 +1,126 @@
+## Basic data retrieval commands
+
+### The `TS.GET` command
+
+`TS.GET` is used to get the sample with the highest timestamp from a given time series. It reports both the timestamp and the value.
+
+```redis TS.GET from bike_sales_3
+TS.GET bike_sales_3 // 500.79 @ timestamp 1648062372210
+```
+
+When `LATEST` is provided as the last argument and with a compaction, `TS.GET` reports the compacted value of the latest (possibly partial) bucket. Without LATEST, `TS.GET` does not report the latest (possibly partial) bucket. When a time series is not a compaction, LATEST is ignored.
```

Comment:
I don't think 'compaction' is explained anywhere.

Reply:
It is... in the Retention and Compaction section.